db/migrate/20220520211359_create_likes.rb

```diff
@@ -0,0 +1,10 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.string :name
+      t.references :user, null: false, foreign_key: true
+      t.references :post, null: false, foreign_key: true
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ]  The likes table should have t.references as `author_id` as a foreign key, and `post_id` as a foreign key.